# Define an action to move the manipulator given a joint configuration
string state
---
---